client/src/components/card/hermit-card-svg.module.css

```diff
@@ -3,7 +3,7 @@
 
 .card {
 	background: white;
-	letter-spacing: 1px;
+	letter-spacing: 0.2rem;
```

Comment:
Please avoid rem units, they aren't used anywhere else so might cause unexpected behavior when someone plays with font-sizes.

Reply:
Ah sorry; I meant to use em rather than rem, but em is the proper unit of size for text adjustments.